### DISCUSSION TITLE
Paste multi-line text into a card to create multiple cards at once.

### DISCUSSION THREAD

**Discussion by erikwestra** [mgmeyers/obsidian-kanban#163] [Ideas & Feature Requests]
Hi there,

Firstly, a huge thank-you to Matthew for creating this awesome plugin -- I use it every day, and love it!

Every day, I find myself having to add maybe a dozen new cards to my Kanban board, and it's kind of slow and klunky to do this at the moment, entering the items one at a time.  Is there any chance you could add a shortcut like Trello has, where you can paste multi-line text when creating a card and it gives you the option of creating a separate card for each line.  That would be a big time-saver when creating lots of cards at once, and I don't think it'd be too hard to implement.

Thanks again for creating use a useful plugin!

 - Erik.

**Comment by mgmeyers** [comment]
Thanks Erik! This should actually work on the latest version:

![Jun-06-2021 09-20-33](https://user-images.githubusercontent.com/2694747/120931922-81d5a600-c6a8-11eb-8ffa-48bb23ad37ce.gif)

Let me know if this isn't working for you.

**Reply by Ray-de-light** [reply]
I also love your Kanban plugin, Mathew. However, pasting in multiple cards doesn't work for me. Is there a special format to use with the pasted text? (Double or single line breaks doesn't get me that result.) Is it included in the main Kanban plugin -- is there a setting or an extra extension needed? Many thanks for any help.

**Comment by erikwestra** [comment]
Wohoo!  Thanks, Matthew, that works perfectly!

**Reply by erikwestra** [reply]
Hi Matthew, this was working until I upgraded the plugin to the latest version, and this feature went away!  Any chance you could restore it, please?

**Reply by mgmeyers** [reply]
Apologies @erikwestra. This should be working again. Obsidian changed some things in 0.12.5 that broke this.

**Reply by slnsysgit** [reply]
now there is also a Split function for multiline cards:
![image](https://github.com/mgmeyers/obsidian-kanban/assets/83591710/9d3ac765-66fb-4c43-afa6-19962199f353)

**Comment by adamhp** [comment]
Is this still expected to work? I'm not able to do it in the current version.
![CleanShot 2023-12-01 at 08 39 50](https://github.com/mgmeyers/obsidian-kanban/assets/7103990/3b0420c9-5b06-420f-acbd-c251a2b6e45c)

Edit: Did a quick work-around by switching to Markdown mode and prepending all my lines with `- [ ]` for now.
